### PR TITLE
Add newcomer map

### PR DESCRIPTION
**Pull request by SURAJITSHAW**
**Description**
Add the contributor journey map Under the Newcomer's Path section on this page (https://layer5.io/community/handbook/about)
This PR fixes #2916

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Zain-Muiz** [comment]
Heyy @SURAJITSHAW ! Good work with the code! You haven't signed of your commit. Its highly recommended to read the contributing.md before you make your contribution. Please follow [these steps](https://github.com/layer5io/layer5/pull/2214/checks?check_run_id=3686442369) to fix it.

**Comment by SURAJITSHAW** [comment]
Sorry @Zain-Muiz  sir, I already read contributing guide but had a little confusion about sign off. Thanks to you I sucessfully signed off the commit. Still if there any problem  with my PR, please let me know.

File: package.json [hunk 1]
```diff
@@ -90,6 +89,7 @@
     "cpx": "^1.5.0",
     "cross-env": "^7.0.0",
     "env-cmd": "^10.1.0",
+    "eslint": "^7.32.0",
```
Comment (Nikhil-Ladha) [inline package.json:92]: This update is not intended.

File: src/sections/Community/Handbook/about.js [hunk 2]
```diff
@@ -33,6 +34,24 @@ const Intro= () => {
             <h2>Newcomer’s Path</h2>
             <p>Thank you for your interest in contributing to Layer5 open source initiatives!</p>
             <p>In combination with the Layer5 Community Guide and Contributing Guide, the following set of instructions are to encourage and support first-time (or near first-time) open-source contributors.</p>
+            <Row className="newcomers-journey">
+            <Col
+              className="heading"
+              sm={12}
+              lg={6}
+              onMouseOver={() => setHover(true)}
+              onMouseOut={() => setHover(false)}
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/about.js:43]: If you are copying the code, always do confirm how the code is working there and what all things should be copied.
There is a `useState` function defined for `setHover` in the code from where the code is taken, and you haven't 
 added that here.

Comment (Nikhil-Ladha) [reply src/sections/Community/Handbook/about.js:43]: Also, where have you redefined the CSS being used there under each classNames ?

File: src/sections/Community/Handbook/about.js [hunk 3]
```diff
@@ -33,6 +34,24 @@ const Intro= () => {
...
+            >
+              <h2>Contributor's Journey</h2>
+              <div className="heading-start">
+                <h5>Start Here</h5>
+                <img className="heading-start__arrow" src={longArrow} />
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/about.js:48]: You haven't imported the image.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Have you confirmed the working of the site locally before sending out the PR?

**Comment by Nikhil-Ladha** [comment]
Also, the wrong issue number is referenced in the commit-msg. The correct issue is #2196

**Pull request closed without merging**